adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala

```diff
@@ -302,35 +318,37 @@ case class DatasetBoundFeatureRDD private[rdd] (
   lazy val rdd = dataset.rdd.map(_.toAvro)
   protected lazy val optPartitionMap = None
 
-  override def saveAsParquet(filePath: String,
```

Comment:
By moving this to `FeatureRDD`, we're always converting to DF even in the RDD path, is that intentional?